website/blog/2019-09-05-7.6.0.md

```diff
@@ -0,0 +1,98 @@
+  static #loaderInstance = null;
+
+  static get #loader() {
+    if (!this.#loaderInstance) this.#loaderInstance = new Loader();
+    return this.#loaderInstance;
+  }
+
+  status = null;
+  
+  constructor(url) {
+    this.status = Resource.#loader.load(url);
+  }
+}
+```
+
+Thanks to [Tim](https://github.com/tim-mc) (Bloomberg) for implementing this proposal!
+
+You can test this new feature by adding `@babel/plugin-proposal-private-methods` to your config, if you haven't already added it, or by enabling the `stage-3` preset in the online [repl](https://babeljs.io/repl/build/master#?presets=stage-3).
+
+Class private features support is finally complete 😄🎉
+
+<div style="margin: auto; width: fit-content">
+
+| Class Private  | **Instance** | **Static** |
+|----------------|--------------|------------|
+| **Fields** <br/> `class A { #a = 1}`    | [7.0.0](https://babeljs.io/blog/2018/08/27/7.0.0#tc39-proposals-https-githubcom-tc39-proposals-support) | [7.1.0](https://babeljs.io/blog/2018/09/17/7.1.0#private-static-fields-stage-3) | 
+| **Methods** <br/> `class A { #a() {} }`   | [7.2.0](https://babeljs.io/blog/2018/12/03/7.2.0#private-instance-methods-8654-https-githubcom-babel-babel-pull-8654) | [7.4.0](https://babeljs.io/blog/2019/03/19/7.4.0#static-private-methods-9446-https-githubcom-babel-babel-pull-9446) |
+| **Accessors** <br/> `class A { get #a() {} }` | [7.3.0](https://babeljs.io/blog/2019/01/21/7.3.0#private-instance-accessors-getters-and-setters-9101-https-githubcom-babel-babel-pull-9101) | 7.6.0 |
+
+</div>
+
+<div style="margin: auto; width: fit-content">
+
+![It's time to party!](https://media.giphy.com/media/3KC2jD2QcBOSc/source.gif)
+
+</div>
+
+### V8 intrinsic runtime functions parsing ([#10148](https://github.com/babel/babel/pull/10148))
+
+> :warning: This is a non-standard extension to the language, which can only be used in V8 when enabled using the `--allow-natives-syntax` compiler flag.
+
+V8, the JavaScript engine used by Node.js and Chromium-based browsers, exposes different native functions internally used when executing your code. They can be useful, for example, to understand how much your code is optimized (or deoptimized), to inspect some JS values or to call some routines used in the ECMAScript specification.
+
+These native functions have a different syntax than normal functions: their name always start with `%`. Also, they can only be directly called and never used as normal values (you can't, for example, assign them to another variable).
+
+```javascript=
+function fn() { /* ... */ }
+
+const status = %GetOptimizationStatus(fn);
+
+if (status === 2) {
+  console.log("The function is not optimized!");
+}
+```
+
+You can find the whole list of existing native V8 functions in [V8's source code](https://github.com/v8/v8/blob/master/src/runtime/runtime.h).
+
+You can enable this syntax in `@babel/parser` by using the `v8intrinsic` plugin:
+
+```javascript=
+const { parse } = require("@babel/parser");
```

Comment:
Should we offer `@babel/syntax-v8-intrinsics`? It should be more convenient for end users.